Example/ios/ReanimatedExample.xcodeproj/project.pbxproj

```diff
@@ -653,6 +654,13 @@
 					"-DFOLLY_MOBILE=1",
 					"-DFOLLY_USE_LIBCPP=1",
 				);
+				OTHER_LDFLAGS = (
+					"$(inherited)",
+					"-Wl",
+					"-ld_classic",
+					" ",
+					"-Wl -ld_classic ",
```

Comment:
Just to be sure: Are these changes auto-generated or you edited this file manually?

Reply:
Yes, these changes were auto-generated during `pod install` when I run it again. For some reason, it didn't work for me before and this was the reason why CIs were failing. I decided to run `rm -rf Pods Podfile.lock build` and `pod install` and then I noticed that this file has been changed, so I pushed the changes.